content/v2.0/get-started.md

```diff
@@ -8,13 +8,161 @@ weight: 1
 v2.0/tags: [get-started, install]
 ---
 
-Get started with InfluxDB OSS v2.0 by downloading InfluxDB, installing the necessary
-executables, and running the initial setup process.
+{{< cloud-name >}} is a fully managed and hosted, version of the InfluxDB 2.0 time series data platform.
```

Comment:
@pierwill, I might start with both links at top, so folks can easily get to OSS

To get started, select the InfluxDB product you’re using:
- Cloud 2.0
- OSS 2.0 (open-source)

Reply:
What do we think of having a note here?

Reply:
@pierwill in addition to having two links at the top or instead of having two links at top? 
...as an OSS user, if I click Get started, and only see "Start using InfluxDB Cloud 2.0" ...